airflow/migrations/versions/b3b105409875_add_root_dag_id_to_dag.py

```diff
@@ -27,7 +27,7 @@
 
 # revision identifiers, used by Alembic.
 revision = 'b3b105409875'
-down_revision = 'd38e04c12aa2'
+down_revision = ('d38e04c12aa2', 'b0125267960b')
```

Comment:
Haha I didnt even know that is possible :-)

Reply:
But can't we adjust the head of AIP-24 btw? That would not make the double reference required?

Reply:
If we change the head of AIP-24, then we detach another head.

Reply:
The alembic migrations are a DAG as well :-)